### ISSUE TITLE
Black Screens After Using

### ISSUE THREAD

**Issue by warriorpoetex** [Rem0o/FanControl.Releases#2283]
Cleaned my PC of ASUS software, rebooted and all worked fine. In fact, rebooted many times ti clean Nalmirc(sp*) off my PC.

Installed Fan Control, set for Curve and rebooted. Didnt auto start. So set the timer for 30 sec after login and rebooted.

PC now fails to even post to BIOS. I have tried clearing the CMOS, everything. This is all new hardware that has been stress tested months ago and was working great until this.

What is going on?

Specs:
Intel i914900K
Asus z790 F Strix
Asus 4070ti
64g RAM

Please, anyone, help.

**Comment by warriorpoetex** [comment]
After diagnosing ... I remembered when updating NVIDIA drivers sometimes, the MONITORS don't activate properly. So, after needlessly checking internals- I unplugged the monitors from the power and cables, and plugged them back in. VIOLA. Works. 

Now, unsure if this software SOMEONE triggered this by could have been happenstance. So if you get black screens, try resetting the monitors as I have suggested.
